k8s-logging-cfg.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logging-cfg
+  namespace: default
+data:
+  filebeat.yml: |
+    filebeat.registry_file: /var/log/containers/filebeat_registry
+
+    filebeat.prospectors:
+    - input_type: log
+      paths:
+        - "/var/lib/docker/containers/*/*.log"
```

Comment:
This needs to be `/var/log/containers/*.log` to get the metadata from filenames.

Reply:
done